int64 object_id
int64 height
string object_type